src/Sulu/Bundle/AdminBundle/Resources/js/components/DatePicker/tests/DatePicker.test.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
```

Comment:
We also use flow in tests, expect for when we are doing something it is not capable of recognizing.